app/main.py

```diff
@@ -1,5 +1,65 @@
+from __future__ import annotations
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
+        self.weight = weight
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
         self.weight = weight
+        self.coords = coords
+        if self.coords is None:
+            self.coords = [0, 0]
```

Comment:
```suggestion
        self.coords = coords or [0,0]
```
apply the same for another case

Reply:
it works, but... how?